### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -2526,3 +2526,46 @@
...
+490124:490125,{ bonus bFixedCast,-200; if (BaseLevel >= 105) { bonus bMatkRate,1; bonus bVariableCastrate,-2; } if (BaseLevel >= 110) { bonus bMatkRate,1; bonus bVariableCastrate,-3; } if (BaseLevel >= 120) { bonus bMatkRate,1; bonus bVariableCastrate,-5; } }
+490126:490127,{ bonus bFixedCast,-200; bonus bAspdRate,10; if (BaseLevel >= 105) { bonus2 bAddClass,Class_All,2; } if (BaseLevel >= 110) { bonus bShortAtkRate,5; bonus bLongAtkRate,5; } if (BaseLevel >= 120) { bonus bCritAtkRate,5; } }
+490128:490129,{ bonus bFixedCast,-200; bonus bVariableCastrate,-10; if (BaseLevel >= 105) { bonus bMatkRate,2; } if (BaseLevel >= 110) { bonus2 bMagicAtkEle,Ele_All,5; } if (BaseLevel >= 120) { bonus2 bMagicAddEle,Ele_All,5; } }
+15362:4123,{ bonus3 bAddEffWhenHit,Eff_Stun,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Freeze,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Stone,5000,ATF_MAGIC; }
+15362:4381,{ bonus bDelayrate,-5; bonus bAspd,1; }
+15367:4141,{ .@r = min(14,getequiprefinerycnt(EQI_ARMOR)); bonus bMaxHPrate,3*(.@r/2); }
+15367:4363,{ .@r = getrefine(); if (.@r>=10) { bonus bMaxHPrate,min(14,.@r)-5; } }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2536]: For me the hp max increases from (refine-5)*(refine-5) with refine up to 14
```suggestion
15367:4363,{ .@r = getrefine(); if (.@r>=10) { .@r = min(14,.@r)-5; bonus bMaxHPrate,pow(.@r,2); } }
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -96516,6 +96677,45 @@ Body:
...
+      if (getskilllv("SN_SHARPSHOOTING") == 5)
+         bonus2 bSkillAtk,"SN_SHARPSHOOTING",60;
+      if (getskilllv("HT_BLITZBEAT") == 5)
+         bonus2 bSkillAtk,"HT_BLITZBEAT",15;
+      if (Class == Job_Ranger) {
+         bonus bMaxHPrate,5;
+         bonus bLongAtkDef,-20;
+         bonus bLongAtkDef,-15;
+         if (.@r>=7)
+            bonus bMaxHPrate,5;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:96707]:
```suggestion
         bonus bLongAtkDef,20;
         if (.@r>=7) {
            bonus bLongAtkDef,15;
            bonus bMaxHPrate,5;
         }
```

File: db/re/item_combo_db.txt [hunk 3]
```diff
@@ -2526,3 +2526,46 @@
...
+19435:4001,{ bonus2 bResEff,Eff_Freeze,10000; }
+19217:4011,{ bonus bMaxHPrate,12; }
+19436:4145,{ bonus bVariableCastrate,30; }
+19436:4374,{ bonus2 bIgnoreMdefClassRate,Class_Boss,70; bonus2 bMagicAtkEle,Ele_Holy,30; bonus2 bMagicAtkEle,Ele_Neutral,30; }
+19436:32246,{ bonus bVariableCastrate,-20; }
+19443:4143,{ if (BaseLevel >= 100) { bonus bBaseAtk,30*(readparam(bVit)/10); bonus bMatk,30*(readparam(bVit)/10); bonus bVit,3*(readparam(bLuk)/10); bonus bLuk,3*(readparam(bLuk)/10); } else if (BaseLevel <= 99) { bonus bBaseAtk,10*(readparam(bVit)/10); bonus bMatk,10*(readparam(bVit)/10); bonus bVit,readparam(bLuk)/10; bonus bLuk,readparam(bLuk)/10; } }
+19444:4143,{ if (BaseLevel >= 100) { bonus bBaseAtk,30*(readparam(bVit)/10); bonus bMatk,30*(readparam(bVit)/10); bonus bVit,3*(readparam(bLuk)/10); bonus bLuk,3*(readparam(bLuk)/10); } else if (BaseLevel <= 99) { bonus bBaseAtk,10*(readparam(bVit)/10); bonus bMatk,10*(readparam(bVit)/10); bonus bVit,readparam(bLuk)/10; bonus bLuk,readparam(bLuk)/10; } }
+20816:4045,{ bonus bLongAtkDef,-35; }
+20816:28910,{ bonus bLongAtkDef,-35; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2549]:
```suggestion
20816:4045,{ bonus bLongAtkDef,35; }
20816:28910,{ bonus bLongAtkDef,35; }
```

File: db/re/item_combo_db.txt [hunk 4]
```diff
@@ -2526,3 +2526,46 @@
...
+20905:22167,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus2 bSubRace,RC_All,10; bonus2 bSubRace,RC_Player_Doram,-10; bonus2 bSubRace,RC_Player_Human,-10; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt:2550]: For me the description include the player races in subrace all
```suggestion
20905:22167,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus2 bSubRace,RC_All,10; }
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -131135,6 +131534,26 @@ Body:
...
+      Head_Top: true
+    ArmorLevel: 1
+    EquipLevelMin: 1
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      bonus bUnbreakableHelm;
+      bonus bAllStats,1;
+      bonus bMdef,10;
+      bonus bMaxHPrate,3*(.@r/3);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:131555]:
```suggestion
      bonus bMaxHPrate,3*(.@r/4);
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -131508,6 +131927,32 @@ Body:
...
+      if (.@r>=7)
+         bonus bMaxHPrate,5;
+      if (.@r>=9)
+         bonus2 bSubEle,Ele_All,5;
+      if (.@r>=10)
+         bonus2 bSubEle,Ele_All,5;
+      if (.@r>=11)
+         bonus2 bSubRace,RC_All,5;
+      if (.@r>=12)
+         bonus2 bSubRace,RC_All,5;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:131955]:
```suggestion
      if (.@r>=10)
         bonus2 bSubClass,Class_All,5;
      if (.@r>=11)
         bonus2 bSubSize,Size_All,5;
      if (.@r>=12)
         bonus2 bSubRace,RC_All,5;
```

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:131555]: Thanks

File: db/re/item_combo_db.txt (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_combo_db.txt (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks

File: db/re/item_combo_db.txt (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks

File: db/re/item_db_etc.yml [hunk 7]
```diff
@@ -46773,6 +46857,142 @@ Body:
...
+    Type: Card
+    Weight: 10
+    Locations:
+      Armor: true
+    Flags:
+      DropEffect: CLIENT
+    Script: |
+      .@r = getrefine();
+      bonus bUseSPrate,10;
+      bonus2 bAddRace,RC_All,4*(.@r/3);
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_etc.yml:46994]: Thanks

File: db/re/item_db_etc.yml [hunk 8]
```diff
@@ -10992,13 +10993,11 @@ Body:
       BuyingStore: true
       DropEffect: CLIENT
     Script: |
+      .@r = getrefine();
       if (getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_MACE) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_etc.yml]: Thanks

File: db/re/item_db_etc.yml [hunk 9]
```diff
@@ -10579,8 +10579,9 @@ Body:
     Script: |
       if (getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW) == W_BOW) {
          bonus2 bSkillAtk,"RA_ARROWSTORM",50;
-         bonus bCriticalLong,25+10*(getequiprefinerycnt(EQI_HAND_R)/4);
+         bonus bCriticalLong,25+(getequiprefinerycnt(EQI_HAND_R)/4);
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_etc.yml:10582]: Thanks

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**